chime/views.py

```diff
@@ -38,6 +38,22 @@ def after_request(response):
 def index():
     return view_functions.render_activities_list()
 
+@app.route('/activity', methods=['GET'])
+@log_application_errors
+@login_required
+@lock_on_user
+@synch_required
+def activity():
+    return view_functions.render_activities_list()
+
```

Comment:
Hey Tomas. It was in anticipation of the root url pointing  to the content view  in the near future and I didn't want to forget about this.  Also, it just made more semantic sense to point it towards where we want it to go as opposed to what the homepage is now. 




Feel free to 



- Norris


Typed on a tiny touchscreen.

On Fri, Sep 18, 2015 at 9:58 AM, Thomas Apodaca <notifications@github.com>
wrote:

>> @@ -38,6 +38,22 @@ def after_request(response):
>>  def index():
>>      return view_functions.render_activities_list()
>>  
>> +@app.route('/activity', methods=['GET'])
>> +@log_application_errors
>> +@login_required
>> +@lock_on_user
>> +@synch_required
>> +def activity():
>> +    return view_functions.render_activities_list()
>> +
> Why have this route instead of having Cancel link back to '/'?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/chimecms/chime/pull/534/files#r39876825